### PR TITLE
test-bot: unlink conflict formulae during the test

### PR DESCRIPTION
**Pull request by xu-cheng**
Solve the problem occured in https://github.com/Homebrew/homebrew/pull/35695

### CODE REVIEW

File: Library/Homebrew/cmd/test-bot.rb [hunk 1]
```diff
@@ -445,6 +445,10 @@ def formula formula_name
         test "brew", "test", "--verbose", formula_name if formula.test_defined?
         testable_dependents.each do |dependent|
           unless dependent.installed?
+            conflicts = dependent.conlicts.map { |c| Formulary.factory(c.name) }.select { |f| f.installed? }
```
Comment (mikemcquaid) [inline Library/Homebrew/cmd/test-bot.rb]: Typo `conlicts.map`. Did you test this locally?
Also, should put this before `install` and after `fetch`.

Comment (xu-cheng) [inline Library/Homebrew/cmd/test-bot.rb]: I haven't yet test it as I don't know which command should be invoked in `test-bot`. I tried `brew test-bot --ci-pr --skip-setup <pr-url>`. But it didn't the real test at all.

Comment (mikemcquaid) [inline Library/Homebrew/cmd/test-bot.rb]: `brew test-bot --skip-setup libffi` should be enough to reproduce/test this.

Comment (xu-cheng) [inline Library/Homebrew/cmd/test-bot.rb]: Sadly, it doesn't. As shown in [CI bot](http://bot.brew.sh/job/Homebrew%20Pull%20Requests/19575/version=yosemite/console), this problem cannot be reproduced on Yosemite.

I'm searching for another formula to test this.

Comment (mikemcquaid) [inline Library/Homebrew/cmd/test-bot.rb]: I'm building a bottle for `rakuda-star` on yosemite which will help.

**Pull request closed without merging**